lib/ansible/modules/commands/psexec.py

```diff
@@ -0,0 +1,527 @@
+    description:
+    - Will run the command with Administrative rights when I(process_username)
+      is defined.
+    - This only affects the process if C(process_username) is set, if it is not
+      then the process is run under the I(connection_username) account and
+      already has an elevated token to the process.
+    - Cannot be C(yes) when I(limited) is also C(yes).
+    type: bool
+    default: 'no'
+  limited:
+    description:
+    - Will run the command with limited rights similar to running a process
+      normally when UAC is turned on.
+    - This only affect the process if C(process_username) is set, if it is not
+      then the process is runder the I(connection_username) account and cannot
+      be run under a limited token.
+    - Cannot be C(yes) when I(elevated) is also C(yes).
+    type: bool
+    default: 'no'
+  use_system:
+    description:
+    - Runs the process under the local SYSTEM account instead of a normal user.
+    - This option overrides C(process_username) if set to C(yes).
+    - The process is run with the highest rights available to a process.
+    - This can be dangerous if used incorrectly.
+    type: bool
+    default: 'no'
+  interactive:
+    description:
+    - Will run the process as an interactive process that shows a process
+      Window of the Windows session specified by I(interactive_session).
+    - The I(stdout) and I(stderr) return values will be null when this is set
+      to C(yes).
+    - The I(stdin) option does not work with this type of process.
+    type: bool
+    default: 'no'
+  interactive_session:
+    description:
+    - The Windows session ID to use when displaying the interactive process on
+      the remote Windows host.
+    - This is only valid when I(interactive) is C(yes).
+    - The default is C(0) which is the console session of the Windows host.
+    default: 0
+  priority:
+    description:
+    - Set the command's priority on the Windows host.
+    - See U(https://msdn.microsoft.com/en-us/library/windows/desktop/ms683211.aspx)
+      for more details.
+    choices:
+    - above_normal
+    - below_normal
+    - high
+    - idle
+    - normal
+    - realtime
+    default: normal
+  show_ui_on_win_logon:
+    description:
+    - Shows the process UI on the Winlogon secure desktop when I(use_system) is
+      C(yes).
+    type: bool
+    default: 'no'
+  process_timeout:
+    description:
+    - The timeout in seconds that is placed upon the running process.
+    - A value of C(0) means no timeout.
+    default: 0
+  stdin:
+    description:
+    - Data to send on the stdin pipe once the process has started.
+    - This option does not do anything when I(interactive) or I(asynchronous)
+      is C(yes).
+requirements:
+- pypsexec
+- smbprotocol[kerberos] for Kerberos authentication
+notes:
+- This module requires the Windows host to have SMB setup and the port 445
+  opened on the firewall.
+- This module will wait until the end process is finished unless
```

Comment:
s/end process/process/ (or are you trying to describe the last process in a chain?)